src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -268,23 +36,6 @@ const customHTMLElementModels = {
     }),
 };
 
-// Define the custom renderer components
-const renderers = {
-    a: AnchorRenderer,
-    code: CodeRenderer,
-    img: ImgRenderer,
-    edited: withLocalize(EditedRenderer),
-};
-
-const renderersProps = {
```

Comment:
Note: I removed this entirely. Pretty sure it's not needed but this is the only change I wasn't sure about.